.gitignore

```
@@ -25,6 +25,7 @@
 !/.vscode/tasks.json
 !/.yamllint
 !/docs/**/*.md
+!/docs/images/**
```

I think this targets one or more nested _folders_. How would you feel about something like

```suggestion
!/docs/images/*.png
!/docs/images/*.svg
```